src/wasm.h

```diff
@@ -1444,12 +1444,13 @@ class Import {
     Global = 3,
   };
 
-  Import() : functionType(nullptr), globalType(none) {}
+  Import() : functionType(nullptr), globalType(none), globalMutable(false) {}
 
   Name name, module, base; // name = module.base
   Kind kind;
   FunctionType* functionType; // for Function imports
   WasmType globalType; // for Global imports
+  bool globalMutable; // for Global imports
```

Comment:
OK yeah; I was going on the design doc which says that mutable globals can't be exported but doesn't say anything about imports. For now though I think I like leaving the restriction until after refactoring; i'll remove that.